docs/source/features.rst

```diff
@@ -16,30 +16,900 @@ Text
  Supported Label Types
 ***********************
 
-Classification
-==============
+TensorBay supports multiple types of labels.
+
+Each :class:`~tensorbay.dataset.data.Data` object
+can have multiple types of :class:`labels<tensorbay.dataset.data.Labels>`.
+
+And each type of :class:`labels<tensorbay.dataset.data.Labels>` is supported with a specific label class,
+and has a corresponding subcatalog class.
+
+.. table:: supported label types
+   :widths: auto
+
+   ==============================  ==================================================  =============================================================
+   supported label types           label classes                                       subcatalog classes
+   ==============================  ==================================================  =============================================================
+   :ref:`features:Classification`  :class:`~tensorbay.label.label.Classification`      :class:`~tensorbay.label.subcatalog.ClassificationSubcatalog`
+   :ref:`features:Box2D`           :class:`~tensorbay.label.label.LabeledBox2D`        :class:`~tensorbay.label.subcatalog.Box2DSubcatalog`
+   :ref:`features:Box3D`           :class:`~tensorbay.label.label.LabeledBox3D`        :class:`~tensorbay.label.subcatalog.Box3DSubcatalog`
+   :ref:`features:Keypoints2D`     :class:`~tensorbay.label.label.LabeledKeypoints2D`  :class:`~tensorbay.label.subcatalog.Keypoints2DSubcatalog`
+   :ref:`features:Sentence`        :class:`~tensorbay.label.label.LabeledSentence`     :class:`~tensorbay.label.subcatalog.SentenceSubcatalog`
+   ==============================  ==================================================  =============================================================
+
+Common label features
```

Comment:
I don't think "features" is a proper word here.
Because the name of this doc section is also "features", and obviously two "features" has different meaning.
So~ find another word.